src/js/index.js

```diff
@@ -9,92 +11,437 @@ const deepFreeze = obj => {
   return Object.freeze(obj);
 };
 
-const accentColors = ['#2AD2C9', '#614767', '#ff8d6d'];
-const neutralColors = ['#425563', '#5F7A76', '#80746E', '#767676'];
-const statusColors = {
-  critical: '#F04953',
-  error: '#F04953',
-  warning: '#FFD144',
-  ok: '#01a982',
-  unknown: '#CCCCCC',
-  disabled: '#CCCCCC',
-};
-
-const colors = {
-  brand: '#01a982',
-  focus: accentColors[0],
-};
-
-const colorArray = (array, prefix) =>
-  array.forEach((color, index) => {
-    colors[`${prefix}-${index + 1}`] = color;
-  });
-
-colorArray(accentColors, 'accent');
-colorArray(neutralColors, 'neutral');
-Object.keys(statusColors).forEach(color => {
-  colors[`status-${color}`] = statusColors[color];
-});
-
 export const hpe = deepFreeze({
+  // name: 'HPE 2',
+  // rounding: 4,
+  // spacing: 24,
+  // defaultMode: 'light',
   global: {
-    colors,
+    colors: {
+      icon: 'text',
+      brand: 'green!',
+      background: {
+        dark: '#1A1F2B',
+        light: '#FFFFFF',
+      },
+      'background-back': {
+        dark: '#1A1F2B',
+        light: '#EFEFEF',
+      },
+      'background-front': {
+        dark: '#222938',
+        light: '#FFFFFF',
+      },
+      'background-contrast': {
+        dark: '#FFFFFF08',
+        light: '#11111108',
+      },
+      text: {
+        dark: '#EEEEEE',
+        light: '#333333',
+      },
+      'text-strong': {
+        dark: '#FFFFFF',
+        light: '#000000',
+      },
+      'text-weak': {
+        dark: '#CCCCCC',
+        light: '#444444',
+      },
+      'text-xweak': {
+        dark: '#999999',
+        light: '#666666',
+      },
```

Comment:
Could be partially removed/cleaned-up after merging https://github.com/grommet/grommet/pull/3722

Reply:
we agreed to keep it all.